doc/architecture/environmental_effect_model.adoc

```diff
@@ -0,0 +1,9 @@
+= Environmental effect model
+
+Environmental effect models consume sensor-view data and produce sensor-view data.
+Environmental effect models may, for example, include the following:
+
+* Shadows and oclusion
+* Weather effects
+* Physics of a sensor
```

Comment:
What is meant by "Physics of a sensor" ?

Reply:
The actual physics of a sensor, for example, its optics, electronics, and mechanics.  We had discussion about this last year (in October if I remember correctly). So I thought, I may just include it here.